frontend/src/pages/projects/screens/detail/workspaces/WorkspacePackageDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from '@patternfly/react-core';
+import { WorkspaceImage } from './useWorkspaceImage';
+
+type WorkspacePackageDetailsProps = {
+  notebookImage: WorkspaceImage;
+};
+
+const WorkspacePackageDetails: React.FC<WorkspacePackageDetailsProps> = ({
+  notebookImage: { packages },
+}) => {
+  const getVersion = (version?: string, prefix?: string): string => {
```

Comment:
I believe I created some utility functions in spawnerUtils that could handle the dependencies and read it from an ImageVersion, maybe we can reuse that?